web/src/api/stores/file.store.ts

```diff
@@ -23,16 +23,19 @@ export const useFileStore = defineStore("files", () => {
   const legend = useLegend(files);
 
   // Parse the files from a CSV string.
-  function parse(fileData: d3.DSVRowArray, anonymize = false): ObjMap<File> {
-    const randomNameGenerator = (): string =>
-      uniqueNamesGenerator({ dictionaries: [colors, names], length: 2 });
+  function parse(fileData: d3.DSVRowArray): ObjMap<File> {
+    const randomNameGenerator = (): string => uniqueNamesGenerator({
+      dictionaries: [colors, names],

```

Comment:
I would change this to only use the `names` dictionary, since colors in the name are a bit confusing. The length could also be just 1, since there are 4900 names which should be enough.